src/peft/tuners/lycoris_utils.py

```diff
@@ -79,8 +79,7 @@ def __init__(self, base_layer: nn.Module) -> None:
 
     @property
     @abstractmethod
-    def _available_adapters(self) -> set[str]:
-        ...
+    def _available_adapters(self) -> set[str]: ...
```

Comment:
Let's undo all the changes in this file. Make sure that the `ruff --version` you use is `0.4.10`.